### ISSUE TITLE
Angular 6 support

### ISSUE THREAD

**Issue by joeldenning** [CanopyTax/single-spa-angular#45]
single-spa-angular@3.0.0-beta.0 through beta.10 all have had angular 7 in mind. Ideally we would support at least Angular 6, as well (and maybe older versions, too).

<img width="650" alt="Screen Shot 2019-04-04 at 10 35 54 AM" src="https://user-images.githubusercontent.com/5524384/55572710-713ed500-56c5-11e9-8c5d-dc4cc1989b3f.png">

**Comment by joeldenning** [comment]
I'm seeing this with Angular 6:

<img width="341" alt="Screen Shot 2019-04-09 at 10 27 15 AM" src="https://user-images.githubusercontent.com/5524384/55817611-200e5700-5ab2-11e9-83d0-7ed596748ab2.png">

**Comment by joeldenning** [comment]
Turns out the maximum call stack size exceeded error was related to `npm link` stuff -- doesn't happen unless you're attempting local dev with angular 6.

j85's original issue is a real one though. It is related to [`context.targetSpecifier`](https://github.com/CanopyTax/single-spa-angular/blob/86fef3d1a5d9c4f40ca10b0da2e55a209c5f294c/src/builders/single-spa-webpack-builder.ts#L7) not being defined in angular 6. I've got a fix coming for it.
